src/test/meson.build

```diff
@@ -772,12 +772,6 @@ tests += [
          [libdl],
          'ENABLE_NSS', 'manual'],
 
-        [['src/test/test-umount.c',
-          'src/shutdown/umount.c',
-          'src/shutdown/umount.h'],
-         [libshared],
-         [libmount]],
```

Comment:
Same story here. The ugly setup where tests are first built into an array, and then declared in a loop in the root is to build them into the root of the build directory.